### DISCUSSION TITLE
Student Form PHOTOs [ How to be approved ]

### DISCUSSION THREAD

**Discussion by jonahanzen** [community/community#147233] [GitHub Education]
### Hi everyone,

Hello everyone, 
Do you guys have any tips to go through the student form?
I have a good camera, thing is, it doesn't accept file uploads, and it doesn't even change to the rear camera.
I took 3-4 photos of my enrollment, of billings, nothing go through, is there any way so that a human can proccess the form? 

Should i focus more on the date, or some kind of signature? on my name?  
Their auto-reply kind of system after sending the photo is as bad as the form so, not quite sure why it's not going through

**Comment by davevad93** [comment]
Hi @jonahanzen ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), "**Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.**"

Unfortunately the process is automated so I don't think there's any kind of manual review. From my experience, I suggest that the academic proof you submit should include in addition to your full name also the name of your school and a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format proving your current student status.
